packages/t/tsparticles.json

```diff
@@ -26,7 +40,10 @@
       {
         "basePath": "dist",
         "files": [
-          "**/*"
+          "tsparticles.slim.js",
+          "tsparticles.slim.min.js",
+          "tsparticles.js",
+          "tsparticles.min.js"
```

Comment:
```suggestion
          "tsparticles*.js"
```
We can condense this using a [glob](https://www.digitalocean.com/community/tools/glob?c=Y29tbWVudHM9dHJ1ZSZnbG9iPWRpc3QlMkZ0c3BhcnRpY2xlcyUyQS5qcyZtYXRjaGXHK3Rlc3RzPSUyRsUDMEltcG9ydGVkJTIwZnJvbSUyMM1AME5QTSUyMHBhY2thZ8g9TElDRU5TRccOx3dyZcRJLmh0bWzVGXNsaW3THlBsdWdpbnMlMkZBYnNvcmJlcsQMT3B0aW%2FFFkNsYXNz5ACKySDkANLfQ8QMRW51bcwUQ2xpY2tNb2Rl30DIQMgMSW5zdGFuY983%2FwC6xiBSYW5kb21TaXrfTfEAhHPfMN99xSDfd9V3xh%2FRNtRnUOoCNEZBbmltYXRhYmxlQ29sb%2FIBsd9ARkF0dHJhY3TROENvcmXLKyUyRkludGXEK8dP00Xff8h%2FQmFja2dyb3VuZM0N3zzQPE1hc2vREd9E30TGEUNvdvMC%2BdRJ6QESdml0eSUyRuQC8sVhdWJibPICAd9D00NCYXPfR99HxkdEaXb%2FAeDzAeBNb3VzxBfGQ%2FIBEcdBQ2FudmHyAzRVdGlsxEbGH8UO2iRpcmPzAS9TaGFwZURyYXfmAynGJsYQ30tXYXJw%2FwE39gE3RXbkBefFI2lja8UO0UjoBOzoAWP6BOz%2FAWPKGsVHb2xsaWTzANvRcMQpc2lvbtV0%2FwPvyj7yAY%2FfO8Q7b3LmBG9pb%2FIEqekBg8Qn9gHx%2FwGD7QK6Q29ubmXzBJjfRNRETGlua%2FIAjf8BnuoDAcdG8wTd6gDz5Qa4dNNl6QCkQ29udGFpbvsDSdkh%2FwGZRkRlbnNpdHnROOgCOlR5cMcoc3Ryb3nED99k%2FwMeRGl2%2FwMc5wGBRGl29QKm8QCaaXb1AJbqBzNQb2x5Z29u5wWx9ACu5AQa30HfQXJhd1N0cm9r%2FACIRW1pdHT6CAHHH99Bx0HoAQbHE%2F8JNdE%2Bxgv%2FCTP%2FALTLH0xpZt9F30XEH1JhdN9F7wC%2F%2FAlu33PHH%2F8JbNNz9wlq6APO5QLbTGlzdGVu9ACg%2FwMq7gMqxgnRROcD80ZyYW1lTWFuYWf%2FA%2FbYaOgDaEdyYWLYZf8E4cc%2B9Avs%2FwCC8QCC9gVo30buATBI5Al8%2FwRcx3LFK%2F8DL%2FYLsMV%2BZmFjxUdJ%2Fwzr30fRR8cg307fTslO%2FwzG31HTUdVL1zXsBmRJ%2Fwxo30THRPcMbNc87QwoSf8MKd1A5wX2y0T%2FDC3fSP8MMewDf81KSULENPkEEM8s%2FwxGzy%2F3A5nfR99H0Ef%2FDJHfS81L%2BwyVzkfGL%2BgCSURhdGHfOM04clByb2Nlc3NQYXJhbd852jlPYmr%2FAPf1AvLlDMPEIUNoYeQBC2VyxRLfS%2FkBQukFJ0n7DKDXTO0DifsMBvUBEv8D895q%2FAw110P5Ajv%2FDBbfSNFI%2FQwazkrpC5%2FNVN880Dz3CHrXNW9yZGluYXRl%2FAdU%2BgF6%2BAwW1W1EaW3EL%2F8Bov8Cm%2BYCm%2FkMHf8LjNFi%2FwvR30XLRf8L1fcKac5I%2FwvZ30XORf8Lat9Jy0n%2FCGzfTM5M%2Fwu630nLSf8IYv8CRugCRv4LOc95eHRlcm5hbMhO%2FwSr%2FwQgxRj1C1HfRdVF%2FwsU3krqAQ%2F7Cxj1ARNIc2zfKcop%2FwdPyCpJbWFn%2FwHZ%2FwcFxUL%2FBwHPdW5mZeUMr8U%2FyA3fPt8%2BU3T0AMj%2FBFjRW0lubGlu%2FwDI%2BgHZSf8GRfUBgPoC3tUw5AJ6%2FwCg9QGvxDjJQf8Cu99CxQlTaGFkb%2FIFyd9I0khUcmlhbmfzCsr%2FAbvQYklMb2NhbFN2Z9FJ7xZx5QKv9xZw2Fz4BAvECfQK8d9I1Uj%2FBRbIQ%2BUP3vULLtd17Qg%2BTW92%2FwFq%2FwFq2UXoD8hEaXLmA5HEDWluZGV42SveHugBHd4mU3RhdHXmEH7eKegUlNYm1hb%2BHRbfNPQUE98z7wGA3jboEtjWHsYW5R4x0Bv%2FEkHKGuoFbf8SRuoSAOsFmv8F1986%2FwXT%2FQKuzFf1Bc%2FYQ%2BgBccY3QXJyYW5nZeQgI%2F8Dm%2BcDGE5vaf8PR%2FcDl8U530LfQs1CRGVsYfIGf99H0kfmEqb%2FAPzsAPxWYWx1%2Fxsw8AJw%2FxUn%2FRRf%2Fgdu6gHNyyZEZXT%2FDmj4ARtPcGFjx3fHC99G30bIRv8PRd9P0U%2F3AbTXTEnGGExvYfQco80ry0L4DKDeZd8w0jD%2FEJ7%2FC2TpC2TkEelsbGHyBSP1AsjoAV7fLs8u9gtF3TP%2FAf%2F%2FAf%2FnBZ3JDd98z3xz%2Fwp12TnkDgX%2FALL%2FALLJDU51bfQXntZ%2F9hFg%2FwYK0kfKJt9M30zOTP8C3%2FwBD%2BkNRMdP%2Fw1G%2FwpO6ApOUHVzaN9F30UyRklSZW30Cef1AaBSZXB1bP8Gud9030jfSNZI%2FxbW30zOTP8W18pIUmf5G7PRKf8LxPYCNFJvdGF0xU%2FGCt9E30TGCv8D1N5NSf8OC9s76QMA9gL59QE46w3P33Pfc8QJ5Qiy%2FwU%2ByXlp%2BiQN0ipDb3X%2FF8b4AKFpesVS%2FxNx30DGQP8B2N9Jy0n%2FCFf%2FA03pA01TbP8CKP8BtUlTdGH%2FGWzbRkn%2FFjLwBfzNSUlTdmdQYXT%2FBVnXeVRyYWnyE7rfOsQMVHdpbmvmCzTHC99G30bIRv8C%2BMxJScUs9gmN1jBlbG%2F1CyDvEYlMaW74A8j%2FDqPyElD%2FDEL0JV%2FoEof%2FExDfPv8TDN9Ey0T5EwjnAQz3C6T%2FCLjNfPkTIm1h8wlexRX1EEH%2FDZztA%2FP2Etr%2FARpG%2FxKn%2FwDr2EH1EqPEKskR2TPeJvICr8Qw%2FwGt8xB4xDb%2FJ8f2ATDoD7j%2FEEDfPv8QPN9DykP%2FBi3YROoOzf8PYd9Cy0L6BwDzFEDQNMYb%2FwCF%2FwCF%2Bw%2BTyj7KUv8MS9Q23yzNLPYFxfAVcU91dP0p3tgg%2FwQX7g%2BX%2Fw%2BW6QFS3yD%2FBFz0AaDfOt86%2FA68cGrSVOgVpucD%2B9ogb2nzCi%2FvBzLnBDb3BzXcLfUNC%2F8FUe4BCPsPPOgCGcsk%2FxVC233NRfcrBNNx%2BQU732jEDvcQX9Nm1WLJKOYkWeYkZ90q6BiM3zL%2FA1TwB2v1D%2FfoAmBRdWFkVHJl0mLIIVLkAaraHmVjdPYIn%2BgAzFJlY3Vyc2l25iXSYfwK7v8Ay8VN9RB930PPQ%2F8QTN9Ey0T%2FEEjfSMpI%2FzOw9grq6CvOx0T5CyxSZXRpbvwQUfYE1OkQCv8QTd9AyAn%2FB3HwCX3GNf8Jf98o%2FwCmxQz%2FC6HbN%2BUOd%2FYOcugC%2BsUe9R2SziLlBYlGdW7mAP7RLekIksQt%2FwQ%2Fy1fVKskiaeQDtE9yTXVsdGlw%2Fza89QEC5hAE%2FxCM3zz8EIjzCYjNMfcJhfAItMQ0%2FwiV%2FwCl%2FxDk%2FwPp6ACH9BDg6AVEU3Bh5AUZR3Jp8izr7we8U3F1YXL4Dx3QLHRhctcq8QIxdGFydOUPz%2F8GbfcBLPYRP%2B8AkFRleHT%2FE5%2FYYfYRJ9Bg5w8p12TrPXvdHG3zB%2FLMIPUO%2BtEh1Eb%2FAObrEY7%2FEdPfQstC9xHP%2Fw5w5QIY%2FwaSxAtVcGRh9CAD6ALZ7zgW5z7pLmpzb27HE1JFQURNRS5tZPgAoP89gskgLmQudPkA5f8%2Bu%2Bc1ht9C%2Bj693zn%2FAMDxMfnfT%2FM%2Bwd8y%2FwCB6zIp30nSe8Yf0zj%2FAqrxPsffQtRC5zJd0jr%2FCTvVR%2F8Ag%2BoAg%2Fg%2Bzd8%2B0T72Ps%2FfRt9G7TKg30v%2FBrHnMkbfRd9F6zJE30nfSeoyQv8B7v8%2B2eU%2B2dlD5z7b0iD0Pt3bJuY%2B39Ih%2FD7h30%2FmPuP%2FAUX%2FEVvsPuXSSvo%2B5%2F8Bc%2F8DYeo%2B6d90yCvpPuv%2FAO7zBBPJQPQBn9898D7v0kHzPvHfZv8C1ugy3d9G30btMtv%2FAaz%2FPvfnPvf8APvoPvnTaekAquo%2B2tMm8D79%2FwD98wGp6DLp0jr8PwHfaP8DPOo%2FA9p28D8F2irwPwf8B3f%2FE4LoPwnfQ99D6zM23En%2FPaHHH99D%2Fz8P%2FwmJ%2Bz8R3zf%2FALrpM6nfR99H6TNb30fxPxffMN936TOJ30fQd%2FkJwPc%2FHf8DRv8DRscJ00b0PyHfbNds7T8j2Wn%2FBQ%2FrPyX%2FAIb%2FAIbqNBPfSNdI9D8p%2FwSC7T8r%2FwvR%2Fz5H6z8t30nfSfI%2FL99Q31D1PzHfU99TylPXTdc37Aaa8T8130bYRvkM2tc%2B%2BT8530LUQvc%2FO99K30r%2FDJ3rA5nNTOg%2FP98uyS7%2FDLTRMfcDtd9J30nSSf8NAd9Nz039DQXOSfM%2FSd86zjruP0vfO9s75D9N%2FwD%2F%2FyXe7z9P3036AUz1P1HfTvcDpf0MbPUBGv8EE99u%2Fwyd10X5Ak3%2FDHzfStNK%2FwyAzkz3P13fPtE%2B%2BQi%2B1zfqP2H%2FAPf3AYb6DHrVceo%2FZd9v%2FwKxx035DIH%2FC%2BrRZP8MMd9HzUf%2FDDXEKP8%2BSsQN%2Fww530fQR%2F8Lxt9LzUv%2FCK7fTtBO%2FwwY30vNS%2F8IpP8CVvE%2Fd%2F8C0cl98j95%2FwCE%2FwRAxBj3C6nfR9VH%2Fwtq30z2P3%2F%2FARvIS%2BQ%2Fgd8ryyv%2FB4nKLOY0QP8A0v8HPes%2Fh99Jy3n2P4nfQN9A5zm4%2FwR6%2FwR6xlHlOb7%2FAI76AenvP4%2F%2FAYzKdvsC9tUy5T%2BT%2FwCm9gG97z%2BV30TfROwsi99K30ruLJH%2FAcn%2FAcnFUOgsedJL%2Bz%2Bd%2FwDE%2FwQtxBjECfYLSd9K1Ur%2FBUDKReo%2Fo9959wiA5Sy9%2FwF2%2FwF220f7P6naLd8g7z%2Bt2ijxP6%2FaK%2B4%2FsdIo2Bj%2FP7XfNvs%2Ft9w1%2FD%2B50jjuP7vSIOs%2FvdId%2FxLbyhrEDOk%2Fv%2FwCiuwSmPQ%2Fwd883Dz%2FBgv7AszMWfcGB9hF%2Bj%2FH%2FwPB6AM65i9H%2FwO99gO97z%2FL30TfROsvkd9J30nsL5f%2FAQTtAQTmP9HZMvYChP8V2fEB3vw%2F1dJE%2FD%2FX3HL5ASXzP9nfSN9I8z%2Fb31HfUfA%2F3d9Oyk7uP9%2FSN80ty0T6DRDead8y1DL%2FES7%2FB6H0P%2Bf%2FAuDISekwYd8w0DD4C6vdNf8CEf8CEekFzckN%2FwCA8QCA7D%2Fx3zvQO%2B4pv%2F8AuP8AuOgxOf8Ag8dD6D%2F3%2FwX5%2FwkKxUnKJt9O307QTv8C9%2F8NuvU%2F%2Fd9L%2FwqsxU%2FkMAjfR99HxBjnL4T%2FAazIRugvb99433jfSt9K10r%2FF5jfTtBO%2FxeZzErkQAvfK8sr%2Fwwu%2BAJG8UAP30bfRvFAEd9P1k%2FoQBPfPdY970AV%2FwFCx0HtQBffd9937EAZ333JfeRAG98szCzmQB3%2FAKf3AKfsQB%2FfQt9C%2FwXa30vQS%2F8Inf8DaesDaeUvRd9H%2FwHD6kAn30jWSOhAKf8GLP9AK%2F8Fhdl95i8G3zzWPPNAL99I30jwQDH%2FAuTHS%2BxAM98yyTLoQDXSMPpAN9lc%2Fw8d6xLw1Ub0DKjQOckI30DfQP8Tst9G80A%2F0kjnART5DAb%2FCQDtAID7E8rlQDDXF%2FcQy%2F8OEPNASd9E8wEm%2FxNN%2FwD13EP%2FQE%2F2BVX2QFH%2FAsnrQFP%2FAMr8QFX%2FATzzATzuQFffQN9A%2FxDI30XvQFvfRtNG8kBd30TfRP8N0P9AYe023P8Aif8Aif4QGcpAylT%2FDLXPeN8u1i74Bfv4QGvaKtoi%2FwQ990Bv%2FwNq%2FAVB%2FA9%2B%2FwGy3zzfPP8PPuZAedIW8EB72yLlQH3%2FB3rwQH%2FfL88v9w2F%2FwWH7gEU%2FQ%2FG%2F0CF%2BBYA%2BACB9ECH33XvQInfbNZs%2BRDz02rXZskq8UBd2iz6QJH%2FAq%2F%2FB7X3EIvxQJXaI%2BZAl9sg6UCZ0iT5QJv%2FANX%2FANX5ERnfRc9F%2FxDm30bNRv8Q4t9KzEr%2FNXz4C1bxQKXZROdAp%2F8ArfMFBvBAqd9C30L%2FEOP1CdvwQIXaN98q%2FwCsxAz%2FDBXdOetAs9JA8kC13yTwQLfSL%2FJAudokzVvXLPlAvf8BDvQBDutAv98%2B3j7%2FAi3%2FQMP3Cef5QMX%2FAKv%2BAKv%2FCyPORPgED%2FcRhPRAy%2F8ICO9Azd8u7UDP%2FwJH8kDR%2FwD09AE4%2BBHp%2BkDV32XTZfgR0f5A2d9o1GjxQFnfRN9E%2FhH5%2Fw545kBd%2FwZM7UBf0izuQGHEIA%3D%3D).

Reply:
Thanks for the suggestion!